src/main/java/com/mainul35/chatapp/viewmodel/MailVerification.java

```diff
@@ -11,8 +11,6 @@
 
 @Data
```

Comment:
Can we delete this class? It's redundant. It's only being used to validate an `email` field.

Reply:
Yes but it was required, therefore its there.
If we want to use CodeVerification class instead, we will have to define another field "verificationCode" in HTML, and we can't leave that empty. Without putting that field in HTML or leaving it empty will lead it to validation exception, which is not expected. Also, keeping that additional field there may create future vulnerability.